src/Bicep.Core/TypeSystem/Az/AzResourceTypeProvider.cs

```diff
@@ -66,6 +66,8 @@ public class AzResourceTypeProvider : IResourceTypeProvider
 
         public static readonly TypeSymbol Tags = new ObjectType(nameof(Tags), TypeSymbolValidationFlags.Default, Enumerable.Empty<TypeProperty>(), LanguageConstants.String, TypePropertyFlags.None);
 
+        public string Version { get; private set; } = "1.0.0";
```

Comment:
I believe `public string Version { get; } = "1.0.0";` will let you initialize in the constructor.

Reply:
But I might be mixing up with Typescript.